sympy/core/numbers.py

```diff
@@ -35,6 +34,7 @@
 rnd = mlib.round_nearest
 
 _LOG2 = math.log(2)
+_LOG10 = math.log(10)
```

Comment:
Is this used anywhere?

Reply:
I thought I removed that, I must've missed it.
I'll get rid of it.